libs/labelbox/src/labelbox/exceptions.py

```diff
@@ -15,6 +15,9 @@ def __init__(self, message, cause=None):
     def __str__(self):
         return self.message + str(self.args)
 
+class ADVError(LabelboxError):
```

Comment:
nit: Not sure I see the value in surfacing an `ADVError` to the customer as opposed to the more generic error sub-types - it's an implementation detail.